src/paulscode/android/mupen64plusae/GalleryActivity.java

```diff
@@ -639,8 +816,13 @@ private void refreshViews()
         // Refresh the preferences object in case another activity changed the data
         mUserPrefs = new UserPrefs( this );
         
-        // Refresh the action bar
-        if( AppData.IS_HONEYCOMB )
-            invalidateOptionsMenu();
+        // Set the sidebar opacity on the two sidebars
+        mDrawerList.setBackgroundDrawable( new DrawerDrawable(
+                mUserPrefs.displayActionBarTransparency ) );
+        mGameSidebar.setBackgroundDrawable( new DrawerDrawable(
+                mUserPrefs.displayActionBarTransparency ) );
```

Comment:
That's good.  It's nice when you can separate cosmetic and functional commits like this.